pkg/v1/image.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 // Image defines the interface for interacting with an OCI v1 image.
+//go:generate counterfeiter -o fake/FakeImage.go . Image
```

Comment:
I think snake_case is the standard convention for golang files names? Just: `image.go`

Reply:
Fixed.